config/locales/en.yml

```diff
@@ -21,3 +21,4 @@
 
 en:
   hello: "Hello world"
+  badge: "Crachá"
```

Comment:
Unless you changed the default locale to `pt` and renamed this file to `pt.yml`, right?